src/basic_recipes/streamplot.jl

```diff
@@ -114,7 +120,7 @@ function streamplot_impl(CallType, f, limits::Rect{N, T}, resolutionND, stepsize
             pnorm = norm(point)
             push!(arrow_pos, x0)
             push!(arrow_dir, point ./ pnorm)
-            push!(colors, pnorm)
+            push!(colors, color(point))
```

Comment:
This should be a call to some external function, which can test whether the color is a function or not and apply that function appropriately.

Reply:
```suggestion
            push!(colors, color_func(point))